docs/scripts/check-closing-quotes.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
```

Comment:
what do you think about putting this check in a CI?
however, it can give false positives, and the documentation writer can be in pain updating this script. maybe a non-required check which is allowed to fail?

Reply:
I’m not a huge fan of non-required checks. IME they just get ignored. If this script had an easy way to add exceptions, we could just make it a required check.

Reply:
I made an attempt. Do you think this is easy enough?